src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -602,6 +602,19 @@ Value ByteCodeInterpreter::interpret(ExecutionState* state, ByteCodeBlock* byteC
             NEXT_INSTRUCTION();
         }
 
+        DEFINE_OPCODE(JumpIfUndefinedorNull)
+            :
+        {
+            JumpIfUndefinedorNull* code = (JumpIfUndefinedorNull*)programCounter;
+            ASSERT(code->m_jumpPosition != SIZE_MAX);
+            if (!registerFile[code->m_registerIndex].isUndefinedOrNull()) {
+                programCounter = code->m_jumpPosition;
```

Comment:
BTW if this bytecode jumps only when the register value is not undefined nor null, the name of this bytecode should be changed according to its operation such as `JumpIfNotUndefinedNorNull`.

Reply:
One more thing.
It seems that this new bytecode needs jump position alignment like other Jump* bytecodes cases bekiw.
https://github.com/Samsung/escargot/blob/1ab89947116a1e2f53011e828bbcef8cc038c875/src/interpreter/ByteCodeGenerator.cpp#L513-L518

Reply:
Can you elaborate for me? What you mean by needs 'jump position alignment' ?  I used the other bytecode jumps as a template. And im not familiar this part of the code.

Reply:
`m_jumpPosition` should be fixed to indicate the real address of jump target.
You can find that `cd->m_jumpPosition = cd->m_jumpPosition + codeBase;` statement does this job.
When the bytecode is generated for the first time, it only calculates the offset of jump target. 
So we fix it later as the above code part.
Also register index is aligned correctly if it is necessary too.